users/test_views.py

```diff
@@ -1,8 +1,8 @@
 class TestViews:
     def test_profile_view(self, client):
-        response = client.get('/users/profile/')
+        response = client.get('/profile/')
         assert response.status_code == 302
 
     def test_edit_profile_view(self, client):
-        response = client.get('/users/edit-profile/')
+        response = client.get('/profile/edit-profile/')
         assert response.status_code == 302
```

Comment:
Again, you can test both with a single test using parametrize
Something like:

```suggestion
@pytest.mark.parametrize("view", ["", "edit-profile/"])
def test_view(self, client):
        response = client.get(f"/profile/{view}")
        assert response.status_code == 302
```